DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core
+{
+    internal static class Constants
+    {
+        internal const string TagKey = "tag";
+        internal const string MethodKey = "method";
+        internal const string Parameters = "params";
+        internal const string RuleSetting = "setting";
+
+        internal const int AgeStringLength = 3;
```

Comment:
>AgeStringLength [](http://example.com/codeflow?start=27&length=15)

move this variant into the class where using it

Reply:
Fixed it.